.github/dependabot.yml

```diff
@@ -1,20 +1,25 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
 version: 2
 updates:
+
   - package-ecosystem: npm
-    directory: '/'
+    directory: "/"
     schedule:
       interval: monthly
       time: '04:00'
       timezone: Europe/Berlin
-    open-pull-requests-limit: 5
+    open-pull-requests-limit: 10
     assignees:
       - UncleSamSwiss
     versioning-strategy: increase
-  - package-ecosystem: npm
+
+  - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
       interval: monthly
       time: "04:00"
       timezone: Europe/Berlin
-    open-pull-requests-limit: 5
-    versioning-strategy: increase
```

Comment:
Why this line was removed?

Reply:
open-pull-requests-limit: 10  ..... is still present.
versioning-strategy isn't working in package-ecosystem: "github-actions"